src/ProductBundle/Entity/BaseProduct.php

```diff
@@ -845,41 +848,6 @@ public function isPriceIncludingVat()
     }
 
     /**
-     * source : http://snipplr.com/view/22741/slugify-a-string-in-php/.
-     *
-     * @static
-     *
-     * @param  $text
-     *
-     * @return mixed|string
-     */
-    public static function slugify($text)
```

Comment:
@ZRazor All removal must be deprecated on current stable before removing on master.